suggestion.md

```
@@ -139,12 +218,16 @@
 - Audit Logging کامل اضافه شد  
 - Validation های تخصصی پزشکی پیاده‌سازی شد
 - سیستم Notification و هشدارهای بالینی اضافه شد
+- **سیستم تشخیص الگوهای غیرطبیعی پیاده‌سازی شد** ✨
 
 🚀 **سامانه GitDM اکنون:**
 - از نظر امنیتی کاملاً محافظت شده است
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)